cypress/unit/store-getter-getHarmonizedPreview.cy.js

```diff
@@ -0,0 +1,185 @@
+import { getters } from "~/store";
+
+let store;
+let originalValue;
+
+describe("getHarmonizedPreview", () => {
+
+    beforeEach(() => {
+
+        store = {
+
+            state: {
+
+                appSetting: {
+
+                    missingValueLabel: "missing value"
+                },
+
+                dataDictionary: {
+
+                    annotated: {
+
+                        column1: {
+
+                            transformationHeuristic: ""
+                        }
+                    }
+                }
+
+            }
+        };
+    });
+
+    it.only("float transformation", () => {
```

Comment:
Is the `it.only` left over from developing and debugging or did you want to only run float transformation?

Reply:
Yes, this is leftover from debugging. Will delete and commit along with the `console.log` removal.